waspc/data/Generator/templates/react-app/src/auth/helpers/user.ts

```diff
@@ -0,0 +1,14 @@
+import { setAuthToken } from '../../api.js'
+import { invalidateAndRemoveQueries } from '../../operations/resources'
+
+export async function setupAuth({ token }: { token: string }): Promise<void> {
```

Comment:
I wasn't really sure which name to use here, this function is setting up our auth/user/session after login.

- `setupAuth`
- `setupUser`
- `prepareAuth`
- `initSession`

Reply:
Also, if the file should be `helpers/user.ts` or `helpers/auth.ts` 🤷

Reply:
I think `initSession`, `initializeSession`, or `startSession` are the best.